app/routes/api/github/get-getUserInfo.tsx

```diff
@@ -0,0 +1,7 @@
+import { Octokit } from "@octokit/core";
+import { env } from "process";
+const octokit = new Octokit({ USERNAME: env.AUTH0_CLIENT_SECRET });
+
+export const getUserInfo = async (email: string) => {
+  return await octokit.request(`GET /search/users?q=${email}`);
```

Comment:
Do we have to handle an error, e.g. we consumed all of the free intents or something?

Reply:
or time out?

Reply:
nop, I'll check it